tests/contrib/litellm/conftest.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+import pytest
+
+from ddtrace.contrib.internal.litellm.patch import patch
+from ddtrace.contrib.internal.litellm.patch import unpatch
+from ddtrace.trace import Pin
+from tests.contrib.litellm.utils import get_request_vcr
+from tests.utils import DummyTracer
+from tests.utils import DummyWriter
+from tests.utils import override_config
+from tests.utils import override_env
+from tests.utils import override_global_config
+
+
+def default_global_config():
+    return {}
+
+
+@pytest.fixture
+def ddtrace_global_config():
+    return {}
+
+
+@pytest.fixture
+def ddtrace_config_litellm():
+    return {}
+
+
+@pytest.fixture
+def litellm(ddtrace_global_config, ddtrace_config_litellm):
+    global_config = default_global_config()
+    global_config.update(ddtrace_global_config)
+    with override_global_config(global_config):
+        with override_config("litellm", ddtrace_config_litellm):
+            with override_env(
+                dict(
+                    OPENAI_API_KEY=os.getenv("OPENAI_API_KEY", "<not-a-real-key>"),
+                    ANTHROPIC_API_KEY=os.getenv("ANTHROPIC_API_KEY", "<not-a-real-key>"),
+                    COHERE_API_KEY=os.getenv("CO_API_KEY", "gCzwKBz4deJXLFgzcZ6Oe7eA8XRFjh6AwklNP7Lt"),
```

Comment:
this isn't a real key, right? If it is, let's wipe it from cohere and regenerate a new key 😭